src/test/regress/expected/pg15.out

```diff
@@ -1363,14 +1363,166 @@ NOTICE:  renaming the new table to pg15.foreign_table_test
 
 (1 row)
 
+DROP SERVER foreign_server CASCADE;
+NOTICE:  drop cascades to 2 other objects
+\set VERBOSITY default
+--
+-- PG15 allows specifying both the tables and schemas for a publication. For example:
+-- CREATE PUBLICATION pub1 FOR TABLE t1,t2,t3, ALL TABLES IN SCHEMA s1,s2;
+-- ALTER PUBLICATION pub1 ADD TABLE t1,t2,t3, ALL TABLES IN SCHEMA s1,s2;
+-- Adding some simple tests where some tables are distributed
+-- There is a variety of combinations we can try here but I am not including them all
+-- Relevant PG commit: 5a2832465fd8984d089e8c44c094e6900d987fcd
+--
+CREATE SCHEMA pub_test_0;
+CREATE SCHEMA pub_test_1;
+CREATE TABLE pub_test_0.dist_table (a int, b int);
+SELECT create_distributed_table('pub_test_0.dist_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE pub_test_0.ref_table (c int);
+SELECT create_reference_table('pub_test_0.ref_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE pub_test_1.citus_local_table(d int);
+SELECT citus_add_local_table_to_metadata('pub_test_1.citus_local_table');
+ citus_add_local_table_to_metadata
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE TABLE pub_test_0.non_citus_table (e int);
+CREATE TABLE pub_test_1.non_citus_table (e int);
+CREATE PUBLICATION pub1 FOR TABLE pub_test_1.citus_local_table, ALL TABLES IN SCHEMA pub_test_0;
```

Comment:
hmm, Citus does not support reference/distributed(https://github.com/citusdata/citus/issues/6294) /even Citus local tables in publications.

I guess we never throw error if Citus tables are added to publications in any time?

Still, I wonder whether we should add these tests or not. My main concern is that would it be confusing for the users?